app/assignment/css/assignments.css

```diff
@@ -150,6 +153,11 @@ ul.dropdown-menu li a {
 	font-size: 12px;
 }
 
+ul.dropdown-menu li .disabled {
```

Comment:
Please rewrite this in BEM syntax (useful primer below)
http://getbem.com/naming/

Short answer is, utilization of element selectors, and CSS style cascading makes CSS work increasingly complex and more bug prone. This is because they are inherently vague, and it is easy for future updates to inadvertently interact with them.

However, if you are working within a framework that supports CSS encapsulation/isolation (like Angular :D), then as long as your components are kept very small the complexity is kept in check and BEM is not as necessary.

The exception to this rule is when you need to override styling from a third party library (bootstrap) and you can't directly attach classes where you want to, or you need to increase the selector specificity to override highly specific library selectors.

I think you'll probably need some specificity to beat bootstrap, but you can at least encapsulate it within a class to so that we 'opt in' to the styling, rather than it affect all bootstrap dropdowns automatically.